### PR TITLE
Correct method signature in example

### PR DESCRIPTION
**Pull request by MarkSymsCtx**
Signed-off-by: Mark Syms <mark.syms@citrix.com>

### CODE REVIEW

File: python/examples/volume/org.xen.xapi.storage.simple-file/sr.py [hunk 1]
```diff
@@ -73,10 +73,10 @@ def attach(self, dbg, configuration):
             urllib.urlencode(configuration, True),
             None))
 
-    def detach(self, dbg, configuration):
+    def detach(self, dbg, sr):
         """
-        [attach configuration]: attaches the SR to the local host. Once an SR is
-        attached then volumes may be manipulated.
+        [detach sr]: detaches the SR, clearing up any associated resources.
+        Once the SR is detached then volumes may not be manipulated.
```
Comment (lindig) [inline python/examples/volume/org.xen.xapi.storage.simple-file/sr.py]: What does "may" mean in this context? I would expect either "must" or "can't".

Comment (MarkSymsCtx) [reply python/examples/volume/org.xen.xapi.storage.simple-file/sr.py]: The comment is, as it should always have been, a copy of the one here https://github.com/xapi-project/xapi-storage/blob/master/generator/lib/control.ml#L413 on the IDL definition.

Comment (MarkSymsCtx) [reply python/examples/volume/org.xen.xapi.storage.simple-file/sr.py]: but it does indeed mean `cannot`

Comment (MarkSymsCtx) [reply python/examples/volume/org.xen.xapi.storage.simple-file/sr.py]: different context of may which can mean either optional or you cannot, English is not a very precise language

Comment (lindig) [reply python/examples/volume/org.xen.xapi.storage.simple-file/sr.py]: For technical documentation I like what the language used in RFCs: http://microformats.org/wiki/rfc-2119

Comment (MarkSymsCtx) [reply python/examples/volume/org.xen.xapi.storage.simple-file/sr.py]: Agreed, but, I didn't write the original comment, that would probably be JonL. If we change it we should change both. I was just making the example consistent with the IDL.

**Pull request merged**